std_msgs/String task
std_msgs/String req
---
zeabus_example/vision_gate data
